src/Models/Tag.php

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
I'd like to keep this file as it was, as well as removing the IDE helper. It's overkill to me.